test/integration/Exploit1.test.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "chai";
+import { Contract, Signer, utils } from "ethers";
+import { ethers } from "hardhat";
+import ERC20ABI from "../../artifacts/@openzeppelin/contracts/token/ERC20/ERC20.sol/ERC20.json";
+import { DolaFlashMinter, Erc20, Exploit1 } from "../../typechain";
+import {
+  deployContract,
+  DOLA,
+  DOLA_DECIMALS,
+  DOLA_WHALE,
+  resetFork,
+  sudo_AddDolaMinter,
+  sudo_TransferToken,
+} from "../utils/integration";
+
+describe("@EvoPsychotic suggested exploit", () => {
+  let myWallet: Signer;
+  let dolaContract: Erc20;
+  let borrower: Exploit1;
+  let flashMinter: DolaFlashMinter;
+  beforeEach(async () => {
+    await resetFork();
+    [myWallet] = await ethers.getSigners();
+    dolaContract = new Contract(DOLA, ERC20ABI.abi, myWallet) as Erc20;
+    flashMinter = await deployContract("MainnetDolaFlashMinter");
+    borrower = await deployContract("Exploit1", [DOLA, flashMinter.address]);
+    await sudo_TransferToken(DOLA, DOLA_WHALE, utils.parseUnits("100000.0", DOLA_DECIMALS), borrower.address);
+    await sudo_AddDolaMinter(flashMinter.address);
+  });
+
+  describe("Exploit 1", () => {
+    it("Exploit", async () => {
+      const totalSupplyBefore = await dolaContract.totalSupply();
+      const loanAmount = utils.parseUnits("100000000.0", DOLA_DECIMALS);
+      await borrower.borrow(loanAmount);
+      const totalSupplyAfter = await dolaContract.totalSupply();
+      expect(totalSupplyAfter).to.be.equal(totalSupplyBefore);
+    });
```

Comment:
How about adding assertion to check the correct amount of expected fee was deducted, no more and no less.

Reply:
Done